app/components/agenda/agenda-detail/sidebar-item.hbs

```diff
@@ -16,7 +16,7 @@
     </span>
     <div class="vlc-agenda-detail-sidebar__text-container">
       <h4 class="vlc-agenda-detail-sidebar__subject">
-        <pre class="vl-u-text-color-agenda-list">
+        <pre class="vl-u-text-color-agenda-list auk-u-text-pre-line">
```

Comment:
Hier staat nog een vl class

Reply:
Dit is blijkbaar styling toegevoegd binnen de vlc-namespace styling dus daarmee niet mee gerefactored. Vermoedelijk heeft dit dan ook de verkeerde naam (qua namespace) gekregen?